ginga/tests/test_iqcalc.py

```diff
@@ -2,13 +2,16 @@
 
 import numpy as np
 import pytest
+from astropy.utils import minversion
 from numpy.testing import assert_allclose, assert_array_equal
 
 from ginga.AstroImage import AstroImage
 from ginga.util import iqcalc, iqcalc_astropy
 from ginga.util.iqcalc import have_scipy  # noqa
 from ginga.util.iqcalc_astropy import have_photutils  # noqa
 
+ASTROPY_LT_7_0 = not minversion("astropy", "7.0.dev")
```

Comment:
I know, this comes late, but this should check for Astropy 6.1.3, as the Astropy patch was backported and 6.1.3 was just released.

Reply:
Ooops sorry for the oversight! Feel free to PR. I won't be able to get to this until Tue the earliest. It's USA holiday weekend. I apologize for any inconvenience caused.